apps/api/v2/src/modules/users/inputs/create-managed-user.input.ts

```diff
@@ -30,4 +31,9 @@ export class CreateManagedUserInput {
   @CapitalizeTimeZone()
   @ApiProperty({ example: "America/New_York" })
   timeZone?: string;
+
+  @IsEnum(Locales)
+  @IsOptional()
+  @ApiProperty({ example: Locales.EN, enum: Locales })
+  locale?: Locales;
```

Comment:
We can define locale type as:

```
locale!: keyof typeof Locales;
```

Reply:
not sure to understand why ?

Reply:
An example would be when typing a test object in e2e